### PR TITLE
[AIRFLOW-5658] Fix broken links when running behind reverse proxy

### PR DESCRIPTION
**Pull request by osule**
Make sure you have checked _all_ steps below.

### Jira

- [x] My PR addresses the following [Airflow Jira](https://issues.apache.org/jira/browse/AIRFLOW/5658) issues and references them in the PR title. 
  - https://issues.apache.org/jira/browse/AIRFLOW-5658
  - ~In case you are fixing a typo in the documentation you can prepend your commit with \[AIRFLOW-XXX\], code changes always need a Jira issue.~
  - ~In case you are proposing a fundamental code change, you need to create an Airflow Improvement Proposal ([AIP](https://cwiki.apache.org/confluence/display/AIRFLOW/Airflow+Improvements+Proposals)).~
  - ~In case you are adding a dependency, check if the license complies with the [ASF 3rd Party License Policy](https://www.apache.org/legal/resolved.html#category-x).~

### Description

- [x] Here are some details about my PR, including screenshots of any UI changes:

This fixes broken links to Configurations and Versions page when running Airflow behind reverse proxy.

### Tests

- [x] My PR adds the following unit tests __OR__ does not need testing for this extremely good reason: No fundamental changes to code.

### Commits

- [x] My commits all reference Jira issues in their subject lines, and I have squashed multiple commits if they address the same issue. In addition, my commits follow the guidelines from "[How to write a good git commit message](http://chris.beams.io/posts/git-commit/)":
  1. Subject is separated from body by a blank line
  1. Subject is limited to 50 characters (not including Jira issue reference)
  1. Subject does not end with a period
  1. Subject uses the imperative mood ("add", not "adding")
  1. Body wraps at 72 characters
  1. Body explains "what" and "why", not "how"

### Documentation

- [ ] ~In case of new functionality, my PR adds documentation that describes how to use it.~
  - ~All the public functions and the classes in the PR contain docstrings that explain what it does~
  - ~If you implement backwards incompatible changes, please leave a note in the [Updating.md](https://github.com/apache/airflow/blob/master/UPDATING.md) so we can assign it to a appropriate release~

### CODE REVIEW

File: airflow/www/static/css/bootstrap-theme.css [hunk 1]
```diff
@@ -3252,7 +3252,8 @@ tbody.collapse.in {
   content: "\e223";
 }
 .glyphicon-duplicate:before {
-  content: "\e224";
+  font-family: "FontAwesome";
```
Comment (mik-laj) [inline airflow/www/static/css/bootstrap-theme.css]: This is an unrelated change. Can you revert it?  We are working on full migration to font-awesome: https://github.com/apache/airflow/pull/6254

Comment (osule) [reply airflow/www/static/css/bootstrap-theme.css]: Sure, I wasn't aware this was taken care of already in a different PR

**Review by milton0825 — APPROVED** [review]

**Comment by OmerJog** [comment]
Ready to merge?
